<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ldmt_randomSelector</class>
 <widget class="QMainWindow" name="ldmt_randomSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Random Select Tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(68,68,68);
}
QWidget#centralwidget{
background-color: rgb(68,68,68);
}
QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}

QRadioButton{
color:rgb(175,175,175);
}
QCheckBox{
color:rgb(175,175,175);
}

QStatusBar QLabel {
    border: 3px solid white;
}

QStatusBar {
    color: rgb(175,175,175);
}

QStatusBar::item {
    border: 1px solid red;
    border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="1" column="1">
     <widget class="QSlider" name="slider_selectRatio">
      <property name="statusTip">
       <string>Slide to edit ratio.</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="text_selectRatio">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 4px;
border:1px solid rgb(30,30,30);
background-color: rgb(30,30,30, 215);
color:rgb(175,175,175);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Input a count like 5000, or target ratio 50%.</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(200,200,200);</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_randomSelect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Random select from selection by a ratio.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Random Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
